docs/contributor/prdoc.md

```diff
@@ -65,7 +41,39 @@ The list of valid audiences is described and documented in the JSON schema as fo
 
 - `Runtime User`: Anyone using the runtime. This can be a token holder or a dev writing a front end for a chain.
 
-## Tips
+If you have a change that affects multiple audiences, you can either list them all, or write multiple sections and
+re-phrase the changes for each audience.
+
+## Record SemVer Changes
+
+All published crates that got modified need to have an entry in the `crates` section of your `PRDoc`. This entry tells
+the release team how to bump the crate version prior to the next release. It is very important that this information is
+correct, otherwise it could break downstream teams' code.
+
+The bump can either be `major`, `minor` or `patch`. There are exceptions for experimental and private API. These are
```

Comment:
We thought about adding a `none` option for tests/typos etc. Do you think that is useful @bkchr?  
I was convinced in the past, but now i think we can also just do it idiomatically and do patch. I think crates like serde also do patch bumps for typos.

Reply:
`patch` for changing a test file or whatever seems overkill to me. but happy to do either.

Reply:
> I think crates like serde also do patch bumps for typos.

Patch for typos in the docs makes sense, patch for adding/changing a test not that much ;) 

As Morgan is saying `None` being the technical correct solution, but `patch` also works. As long as you explain this, choose whatever you think is the best.

Reply:
Okay since we have the `R0-silent`, we can use the `none` as `R0-silent` label on a per-crate basis.  
Added it now.